app/assets/stylesheets/student_menu.scss

```diff
@@ -1,18 +1,19 @@
 @import 'color';
 
 .student_menu {
-  width: 500px;
+  width: 600px;
```

Comment:
Properties should be ordered height, margin-bottom, margin-left, padding-top, width